src/DynamoConnector.ts

```diff
@@ -67,8 +67,9 @@ export const useDynamoConnector = () => {
         DynamoConnectionState.LOST_CONNECTION,
       ].includes(state.connectionState)
     ) {
-      // @ts-ignore
-      //intervalId = setInterval(() => portDiscovery(), 2000);
+      intervalId = setInterval(() => {
+        portDiscovery();
+      }, 2000);
```

Comment:
Adding this back in seems to fix most issues related to disconnecting?

Reply:
I though this would increase the interference with other calls, but it might be ok :)

Reply:
Make sure to test this without the `ext:daas` feature toggle too 👍